<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chord_window</class>
 <widget class="QWidget" name="Chord_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/icon.png</normaloff>Images/icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
color: rgb(255, 255, 255);
height: 60px;
width: 90px;
border:5px solid rgb(0,0,0);
border-radius:8px;
}

</string>
  </property>
  <widget class="Sound_button" name="Button_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>285</width>
     <height>197</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Sound_button" name="Button_dur">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>285</width>
     <height>197</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Sound_button" name="Button_mol">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>285</width>
     <height>197</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Animated_progressbar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	background_color: rgb(68,76,86);
	border_style: none;
	border_radius: 20px;
}

QProgressBar::chunk{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 154, 147, 255), stop:1 rgba(255, 255, 255, 255));
	margin:1px;
	border_radius: 10px;
	
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>701</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/new_background2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Button_7</zorder>
  <zorder>Button_dur</zorder>
  <zorder>Button_mol</zorder>
  <zorder>progressBar</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sound_button</class>
   <extends>QPushButton</extends>
   <header>Sound_button</header>
  </customwidget>
  <customwidget>
   <class>Animated_progressbar</class>
   <extends>QProgressBar</extends>
   <header>Animated_progressbar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
